libs/perun/components/src/lib/entity-search-select/entity-search-select.component.html

```diff
@@ -22,6 +25,9 @@
       [maxBufferPx]="10 * 48">
       <mat-option *cdkVirtualFor="let entity of filteredEntities | async" [value]="entity">
         {{mainTextFunction(entity)}} <span class="text-muted muted">{{secondaryTextFunction(entity)}}</span>
+        <span *ngIf="displayStatus" class="{{colorByStatus(entity)}}">
+          ({{statusTextFunction(entity)}})
+        </span>
```

Comment:
what about reusing perun-web-apps-group-resource-status component? with this you must  implement another functionality, which is with this on two places
the problem starts if we change name of the status or add/remove status, then you need to change it in two places
also the way how it looks is not nice

Reply:
I already tried to implement it like this, but I afraid that the icon isn't possible to show in closed search select (it display only name of mat-icon). 

![image](https://user-images.githubusercontent.com/70371638/133107532-219e3119-3dc5-41b5-891b-97360bca3c6b.png)
![image](https://user-images.githubusercontent.com/70371638/133107611-a2f1f91d-305c-4e78-af43-c11dc3574de1.png)
![image](https://user-images.githubusercontent.com/70371638/133107657-36631352-0ba8-467e-9d51-7cef1c5c6281.png)